src/transceivercontroller/SimulcastTransceiverController.ts

```diff
@@ -98,11 +98,17 @@ export default class SimulcastTransceiverController extends DefaultTransceiverCo
         direction: 'inactive',
         streams: [this.defaultMediaStream],
       });
+
+      /* istanbul ignore next */
+      if (this.meetingSessionContext?.audioProfile.hasRedundancyEnabled()) {
```

Comment:
Can `audioProfile` be `undefined` as well may not have `hasRedundancyEnabled` implemented?

Reply:
Yes, we should add safe navigation to `audioProfile` as well since it can be `null`.

Reply:
Added safe navigation operators.